procrastinate/sql/migrations/03.00.00_50_post_cancel_notification.sql

```diff
@@ -160,34 +197,44 @@ BEGIN
 END;
 $$;
 
-CREATE FUNCTION procrastinate_trigger_status_events_procedure_update()
+-- Recreate or rename the triggers & their associated functions
+
+ALTER FUNCTION procrastinate_trigger_status_events_procedure_insert
+    RENAME TO procrastinate_trigger_function_status_events_insert_v1;
+
+CREATE TRIGGER procrastinate_trigger_status_events_insert_v1
+    AFTER INSERT ON procrastinate_jobs
+    FOR EACH ROW WHEN ((new.status = 'todo'::procrastinate_job_status_v1))
+    EXECUTE PROCEDURE procrastinate_trigger_function_status_events_insert_v1();
+
+CREATE FUNCTION procrastinate_trigger_function_status_events_update_v2()
```

Comment:
Was it intentional that you renamed `procrastinate_trigger_status_events_procedure_update` to `procrastinate_trigger_function_status_events_update`?